eng/ci/templates/jobs/run-unit-tests-windows.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+- job: RunUnitTests_Windows
+  displayName: '[Windows] Run Unit Tests'
+
+  pool:
+    name: 1es-pool-azfunc
+    image: 1es-windows-2022
+    os: windows
+
+  steps:
+  - task: NuGetAuthenticate@1
```

Comment:
Did you copy this from the host CI files? I meant to remove that, it is leftover from some workaround I had earlier. You shouldn't need this step anywhere, we don't use any authenticated nuget feeds.

Reply:
guilty

Reply:
It was helpful though, reminded me to go clean that up myself lol